### ISSUE TITLE
Display maximum income data according to occupancy

### ISSUE THREAD

**Issue by trevorshannon** [philhawksworth/the-united-effort-orginization#32]
It seems that the maximum allowed income for a given rental unit scales with the number of occupants that will be living in the unit.  This is displayed in the current website, but in a non-ideal way: it's a bit of a side note and the single maximum income value shown is just one of the multiple listed in the side note.

There is probably a better way to show this information.

**Comment by clairehubel** [comment]
Thanks for catching this. Would it work to have a field that records the number of occupants that will be living in a place, and then have that tied to the maximum allowed income, so that the user is not necessarily aware of the calculation, but nevertheless is given accurate info?

**Comment by trevorshannon** [comment]
Yes, I wonder if having an input field in the page that shows all property
rental data would be best.  It does not make sense in the housing search
page, I'm pretty sure.

I'll also have to think about how to display the data for a user who
chooses not to enter in the number of occupants

**Comment by trevorshannon** [comment]
Actually, maybe having a number of occupants field in the search form is necessary in order to filter by income properly.

**Comment by clairehubel** [comment]
I think it likely is. I was just noticing that the AMI varies according the number of people in the household. So to generate an accurate AMI, the size of the household need to be known.

**Comment by trevorshannon** [comment]
Without making any changes and simply printing out the "max income details" like we do on the current website, the new website will have the same functionality as our current one.

This can be improved (as discussed above) but I am wondering, @clairehubel, should this improvement be in time for the 2.0 relaunch or could it come later?  It does seem like a bit of an issue to be unable to filter by max income properly, but at the same time that's how the current site works so...

**Comment by clairehubel** [comment]
I'd say leave as is and work on it post-launch. When we work on the affordable housing db for San Mateo County we can set all this up a little better from the outset 😉
